index.js

```diff
@@ -118,20 +128,23 @@ const parseArgs = (
         if (arg.length > 2) {
           for (let i = 2; i < arg.length; i++) {
             const short = StringPrototypeCharAt(arg, i);
+            // Case of `-o=foo`:
+            if (short === '=') break;
             // Add 'i' to 'pos' such that short options are parsed in order
             // of definition:
             ArrayPrototypeSplice(argv, pos + (i - 1), 0, `-${short}`);
           }
         }
-
+        const suffix = StringPrototypeSlice(arg, 2); // maybe -f=bar.
```

Comment:
I will add a test for this.

Reply:
The suffix code is only for what I think is mistaken processing of `=`, per previous comments.

(On a separate note, we use arg fairly loosely though this code. We could try for some more meaningful names, but I suggest try that is a pure refactor and not when we are changing code.)